### PR TITLE
WIP : API Initialization

### PR DESCRIPTION
**Pull request by iveelsm**
STILL A WORK IN PROGRESS

This is the mockup for the initial API structure for the Viewzr application

### CODE REVIEW

**Comment by iveelsm** [comment]
As it stands, this is technically a complete feature branch since I was able to do a single GET request. But further expansion can be made if desired. If no one has any opinions, I will move on the story branch.

File: api/lib/controllers/about/about.controller.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+var mongoose = require('mongoose');
+
+function getAbout() {
```
Comment (wesharper) [inline api/lib/controllers/about/about.controller.js:3]: What is the about route for?

File: api/lib/models/users/Users.schema.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+'use strict';
+var mongoose = require('mongoose');
+var Schema = mongoose.Schema;
+
+var UsersSchema = {
```
Comment (wesharper) [inline api/lib/models/users/Users.schema.js:5]: Convention singular "UserSchema"

File: api/lib/models/users/Users.schema.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
+'use strict';
+var mongoose = require('mongoose');
+var Schema = mongoose.Schema;
```
Comment (wesharper) [inline api/lib/models/users/Users.schema.js:3]: es6 shorthand const { Schema } = mongoose;

**Pull request closed without merging**